Sources/SwifterSwift/UIKit/UIViewExtensions.swift

```diff
@@ -514,6 +514,13 @@ public extension UIView {
         layer.addSublayer(gradientLayer)
     }
 
+    /// SwifterSwift: Removes the applied blur effect from the `UIView`.
+    func removeBlur() {
+        for subview in subviews where subview is UIVisualEffectView {
```

Comment:
Sorry to add late, but I think we should add this.
```suggestion
        for subview in subviews where subview is UIVisualEffectView, subview.effect is UIBlurEffect {
```

Reply:
```swift
for subview in subviews where subview is UIVisualEffectView, subview.effect is UIBlurEffect { 
```

If used as it is, a compilation error occurs. Therefore, I maintained the same logic but made adjustments by adding an if statement.
```swift
    func removeBlur() {
        for subview in subviews where subview is UIVisualEffectView {
            if let blurView = subview as? UIVisualEffectView, blurView.effect is UIBlurEffect {
                blurView.removeFromSuperview()
            }
        }
    }
```

Reply:
Let's make it nice and functional.
```swift
func removeBlur() {
    subviews
        .lazy
        .compactMap { $0 as? UIVisualEffectView }
        .filter { $0.effect is UIBlurEffect }
        .forEach { $0.removeFromSuperview() }
}
```

Reply:
Nice code! I modified it like that.